### ISSUE TITLE
Swift 2.3 and 3 support

### ISSUE THREAD

**Issue by tiferrei** [amayne/SwiftString#23]
Hello there, 

I would like to know if someone's already working on updating this library, and if not if you intend to, I would also like to help in this.

Thank you,
tiferrei

**Comment by jlaws** [comment]
Would be great to have Swift 3 support now that the GM version of Xcode 8 is out.  Any news on this?

**Comment by tiferrei** [comment]
Yes @amayne and @jlaws, I believe now that Xcode 8 was released with a stable version of Swift 3 it would be a good time to update.

**Comment by matthijs2704** [comment]
https://github.com/iamjono/SwiftString

**Comment by iCodist** [comment]
Swift 3 support +1

**Comment by n13** [comment]
Tried converting it to Swift 3 myself, it's a nightmare. The automatic conversion in XCode 8 totally fails, and pretty much _everything_ having to do with Strings has changed. Would need to basically manually rewrite it.

**Comment by matthijs2704** [comment]
@n13 Why are you still worrying about converting? Just use https://github.com/iamjono/SwiftString

**Comment by n13** [comment]
^^^ Thank you, sir!

**Comment by EddieDow** [comment]
@matthijs2704  Thanks for the upgrading to 3.x

**Comment by abellee** [comment]
@matthijs2704 so I have to install the SwiftString before yours?
after I have installed your repo and I neither haven't seen the SwiftString files nor can't import SwiftString into my class.

**Comment by TimBuckers** [comment]
@abellee I have the same problem after successfully installing the 'String+Extensions', the files are not showing in xcode. Also I can't import the SwiftString in classes, but the files are installed I can find them with the file explorer.

**Comment by aloveric** [comment]
Swfit 3 
+1
